### ISSUE TITLE
Standardize rights metadata

### ISSUE THREAD

**Issue by ryscher** [datadryad/dryad-product-roadmap#3418]
When reviewing our statistics shown at https://commons.datacite.org/repositories/dryad.dryad, and we were surprised to see the Licenses graph showing that 63% of our DOIs have “missing” licenses. I spot checked a number of DOIs, and all of them seem to have entries in the rightsList, although some of them have more complete information than others.

I asked DataCite to clarify what went into this graph, and they responded:
```
1. The rightsURI attribute or rights name matches the format in 
   SPDX: https://spdx.org/licenses/ (i.e. if the rightsURI or rights 
   name matches a SPDX record, the system will normalise it).

2. Any information that has been submitted in the rightsIdentifier 
   element will appear in the license information in Commons regardless 
   of SPDX matching.
```

Therefore, we need to:
- Update the values in the `dcs_rights` table to use the SPDX values in `rights` and `rights_uri`
- Run the rake task `datacite_target:update_dryad` to update identifiers at DataCite
- Walk through the legacy file-level DOIs that are not currently in our database, like 10.5061/DRYAD.20/3 (We may need to query the DataCite API to get these). For each of these DOIs, update the rights metadata to match the new values, but don't change anything else.

**Comment by alinvetian** [comment]
Currently we have 5 different "rights" in our Datacite rights table:
```
#<StashDatacite::Right:0x00007f15f7d9f508 id: nil, rights: "CC0 1.0 Universal (CC0 1.0) Public Domain Dedication", rights_uri: "https://creativecommons.org/publicdomain/zero/1.0/">,
 #<StashDatacite::Right:0x00007f15f7d9f288 id: nil, rights: "Creative Commons Attribution 4.0 International (CC BY 4.0)", rights_uri: "https://creativecommons.org/licenses/by/4.0/">,
 #<StashDatacite::Right:0x00007f15f7d9f008 id: nil, rights: "Terms of use are available at: doi:10.5060/D8PP47", rights_uri: "https://dx.doi.org/10.5060/D8PP47">,
 #<StashDatacite::Right:0x00007f15f7d9ed88 id: nil, rights: "UCSF Datashare Data Use Agreement", rights_uri: "https://dx.doi.org/10.5060/D8TG65">,
 #<StashDatacite::Right:0x00007f15f7d9eb08 id: nil, rights: "Creative Commons Attribution 3.0 International (CC BY 3.0)", rights_uri: "https://creativecommons.org/licenses/by/3.0/">
```

Here are the counts of the resources per each item:
```
"CC0 1.0 Universal (CC0 1.0) Public Domain Dedication"=>209147,
 "Creative Commons Attribution 4.0 International (CC BY 4.0)"=>775,
 "Terms of use are available at: doi:10.5060/D8PP47"=>1,
 "UCSF Datashare Data Use Agreement"=>24,
 "Creative Commons Attribution 3.0 International (CC BY 3.0)"=>1
```

@ryscher We have the related value in SPDX for CC0, what should the values from SPDX that we need to use for the other 4 items?

**Comment by ryscher** [comment]
The other items will not change to SPDX now. I want to review the datasets attached to these, since I thought we had converted all published content to CC0.
